neqo-common/src/datagram.rs

```diff
@@ -53,6 +53,16 @@ impl Datagram {
     pub fn ttl(&self) -> Option<u8> {
         self.ttl
     }
+
+    #[cfg(feature = "udp")]
+    #[must_use]
+    pub(crate) fn into_data(self) -> Vec<u8> {
+        self.d
+    }
```

Comment:
Note that the `udp` feature no longer exists and that the `into_data` function was moved to a `From<Datagram> for Vec<u8>`. See https://github.com/mozilla/neqo/pull/1736.